c10/core/ScalarType.h

```diff
@@ -510,34 +510,61 @@ static inline ScalarType toUnderlying(ScalarType t) {
 
 static inline bool isSignedType(ScalarType t) {
   TORCH_CHECK(!isQIntType(t), "isSignedType not supported for quantized types");
-#define CASE_SIGNED(ctype, name) \
-  case ScalarType::name:         \
-    return std::numeric_limits<ctype>::is_signed;
+
+#define CASE_ISSIGNED(name)     \
+  case ScalarType::name:        \
+    return std::numeric_limits< \
+        ::c10::impl::ScalarTypeToCPPTypeT<ScalarType::name>>::is_signed;
 
   switch (t) {
+    case ScalarType::QInt8:
+    case ScalarType::QUInt8:
+    case ScalarType::QInt32:
+    case ScalarType::QUInt4x2:
+    case ScalarType::QUInt2x4:
+      TORCH_CHECK(false, "isSignedType not supported for quantized types");
```

Comment:
Line 512 has checked the quantization type :D

Reply:
Good point, forgot to remove that one